src/components/DataProductCard.tsx

```diff
@@ -121,30 +30,119 @@ const DataProductCard: React.FC<IProps> = ({
   description,
   data,
 }) => (
-  <Card hideArrow={true} to={url}>
-    <ImageWrapper background={background}>
-      <Image image={image} alt={alt ? alt : `${name} logo`} />
-    </ImageWrapper>
-    <Content>
-      <div>
-        <Title>{name}</Title>
-        {description && <Description>{description}</Description>}
-      </div>
+  <Link
```

Comment:
Instead of wrapping this component with a Link, lets use the `LinkOverlay` component that Chakra offers for this specific scenario https://chakra-ui.com/docs/components/link-overlay/usage

I did a similar implementation in https://github.com/ethereum/ethereum-org-website/pull/7721

Reply:
I already did it.